ros_speech_recognition/euslisp/ros-speech-recognition-client.l

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env roseus
+;; This is the euslisp version of ros_speech_recognition client.
+
+(defclass speech-recognition-client
+    :slots (speech-recog-req speech-recog-res text))
+(defmethod speech-recognition-client
+    (:init ()
+     (ros::roseus-add-srvs "speech_recognition_msgs")
```

Comment:
I don't know why this happens, but this line should be on the top of this script. otherwise,

```lisp
4.irteusgl$ load "package://ros_speech_recognition/euslisp/ros-speech-recognition-client.l"
Call Stack (max depth: 20):
  0: at (apply #'ros::load-org-for-ros ros::fullname args)
  1: at (apply #'ros::load-org-for-ros ros::fullname args)
  2: at (let ((ros::fullname fname)) (when (substringp "package://" fname) (setq ros::fullname (ros::resolve-ros-path fname)) (when ros::*compile-message* (let* ((ros::urlname (url-pathname fname)) (package-name (send ros::urlname :host)) (ros::path-name (format nil "~A_~A" package-name (substitute 95 47 (concatenate string (subseq (send ros::urlname :directory-string) 1) (send ros::urlname :name))))) (ros::old-module (find ros::path-name *loaded-modules* :key #'lisp::load-module-file-name :test #'equal))) (unless ros::old-module (let* ((ros::ppath (unix:getenv "CMAKE_PREFIX_PATH")) (dir (format nil "~A/share/roseus/ros/~A" (subseq ros::ppath 0 (position 58 ros::ppath)) package-name))) (unless (probe-file dir) (unix:mkdir dir)) (compiler:compile-file-if-src-newer ros::fullname (format nil "~A/~A" dir ros::path-name)) (return-from load (load (format nil "~A/~A.so" dir ros::path-name) :entry (format nil "___~A" ros::path-name)))))))) (if (null ros::fullname) (error "file ~s not found" fname)) (apply #'ros::load-org-for-ros ros::fullname args))
  3: at (load "package://ros_speech_recognition/euslisp/ros-speech-recognition-client.l")
  4: at #<compiled-code #X555a73060e18>
/opt/ros/melodic/share/euslisp/jskeus/eus/Linux64/bin/irteusgl 0 error: no such package SPEECH_RECOGNITION_MSGS in (apply #'ros::load-org-for-ros ros::fullname args)
```